File: geek-university/algoritmo-e-logica-de-programacao/c/exercicio_37.c
#include <stdio.h>

int main() {
    float vetor[5];
    int codigo;

    printf("Informe o codigo: ");
    scanf("%d", &codigo);
    if (codigo != 0) {
        for (int i = 0; i < 5; i++) {
            printf("Informe um numero real: ");
            scanf("%f", &vetor[i]);
        }

        if (codigo == 1) {
            for (int i = 0; i < 5; i++) {
                printf("%.2f\n", vetor[i]);
            }
        }

        if (codigo == 2) {
            for (int i = 4; i >= 0; i--) {
                printf("%.2f\n", vetor[i]);
            }
        }
    }
}

File: geek-university-algoritmo-e-logica-de-programacao/java/exercicio_37.java
import java.util.Scanner;

public class exercicio_37 {

    public static void main(String[] args) {
        float[] vetor = new float[5];
        int codigo;
        Scanner teclado = new Scanner(System.in);

        System.out.print("Informe o codigo: ");
        codigo = teclado.nextInt();

        if (codigo != 0 && codigo <= 2) {
            for (int i = 0; i < 5; i++) {
                System.out.print("Informe um numero real: ");
                vetor[i] = teclado.nextFloat();
            }

            if (codigo == 1) {
                for (int i = 0; i < 5; i++) {
                    System.out.println(vetor[i]);
                }
            }

            if (codigo == 2) {
                for (int i = (vetor.length - 1); i >= 0; i--) {
                    System.out.println(vetor[i]);
                }
            }
        }

        teclado.close();
    }

}
